src/hotspot/share/gc/serial/defNewGeneration.cpp

```diff
@@ -94,50 +94,48 @@ class RootScanClosure : public OffHeapScanClosure {
 class CLDScanClosure: public CLDClosure {
 
   class CLDOopClosure : public OffHeapScanClosure {
-    ClassLoaderData* _scanned_cld;
+    DefNewGeneration* _g;
 
     template <typename T>
     void do_oop_work(T* p) {
```

Comment:
Pre-existing: since this method has only a single caller, I suggest inlining it.